services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision-v2.module.ts

```diff
@@ -54,6 +58,9 @@ import { ApplicationDecisionConditionController } from '../application-decision-
       ApplicationDecisionConditionType,
       LinkedResolutionOutcomeType,
       NaruSubtype,
+      ApplicationSubmissionToSubmissionStatus,
+      ApplicationSubmission,
```

Comment:
Do you still need these here? I don't see the AppSubmission repository being used.

Reply:
it is required for the submissionStatusService since it is not in the same module. In part 2 SubmissionStatusService will be a separate module and will be imported instead

Reply:
implemented in this mr instead of the second part